File: src/easy/hours_to_seconds.zig
const expect = @import("std").testing.expect;

pub fn hoursToSeconds(hours: i32) i32 {
    return hours * 3_600;
}
test "integer subtraction" {
    try expect(hoursToSeconds(2) == 7200);
    try expect(hoursToSeconds(10) == 36000);
    try expect(hoursToSeconds(24) == 86400);
    try expect(hoursToSeconds(36) == 129600);
}

File: src/easy/hours_to_seconds.rs
pub fn hours_to_seconds(hours: i32) -> i32 {
    hours * 3_600
}
#[cfg(test)]
mod tests {
    use super::hours_to_seconds;
    #[test]
    fn test_hours_to_seconds() {
        assert_eq!(hours_to_seconds(2), 7200);
        assert_eq!(hours_to_seconds(10), 36000);
        assert_eq!(hours_to_seconds(24), 86400);
        assert_eq!(hours_to_seconds(36), 129600);
    }
}
